### PR TITLE
Adds the Feature: sidebar component

### PR DESCRIPTION
**Pull request by thewahome**
## Overview

Adds a filterable set of categorized sample queries on the sidebar component

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Load the app
* Check the left side of the app and see the sidebar component
* Click on the samples.
* Search for the particular sample you want to fire

### CODE REVIEW

File: src/app/views/sidebar/sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@import '../../../styles/variables';
```
Comment (jobala) [inline src/app/views/sidebar/sidebar.scss]: Implement these styles using css-in-js

**Comment by jobala** [comment]
image

The text inside the rows are not vertically aligned

**Comment by jobala** [comment]
image

The font is in the Sidebar component is different from the rest of the application.

File: src/app/views/sidebar/sidebar.scss (hunk 1 above)
Comment (cwahome) [reply src/app/views/sidebar/sidebar.scss]: Handled in [adds sample queries to the view](https://github.com/microsoftgraph/microsoft-graph-explorer-v2/pull/136)

File: src/app/views/App.tsx [hunk 2]
```diff
@@ -25,7 +26,10 @@ class App extends Component<IAppProps> {
       <FocusTrapZone>
         <div className={`container-fluid ${classes.app}`}>
           <div className='row'>
-            <div className='col-sm-12 col-lg-8 offset-lg-2'>
+            <div className='col-sm-3 col-md-3'>
```
[the remark this answers is not in the document]
Comment (cwahome) [reply src/app/views/App.tsx]: Quite right, it's handled in [adds sample queries to the view](https://github.com/microsoftgraph/microsoft-graph-explorer-v2/pull/136)

File: src/app/views/query-runner/QueryInput.tsx [hunk 3]
```diff
@@ -1,44 +1,61 @@
...
-}: IQueryInputControl) => {
+    const {
+      handleOnRunQuery,
+      handleOnMethodChange,
+      handleOnUrlChange,
+      httpMethods,
+      selectedVerb,
+      sampleUrl,
+      submitting,
+    } = this.props;
```
[the remark this answers is not in the document]
Comment (cwahome) [reply src/app/views/query-runner/QueryInput.tsx]: nice catch

**Comment by cwahome** [comment]
Closing this as it is a duplicate of #136 
Recommendations in the comments have been done in the above pull request

**Pull request closed without merging**